src/Features/CSharp/Portable/CodeFixes/PopulateSwitch/PopulateSwitchCodeFixProvider.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Simplification;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeFixes.PopulateSwitch
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.AddLabelsToSwitch), Shared]
+    [ExtensionOrder(After = PredefinedCodeFixProviderNames.ImplementInterface)]
+    internal partial class PopulateSwitchCodeFixProvider : CodeFixProvider
```

Comment:
I could try, but it is very difficult because they have no shared types.  The `SyntaxKind`s are all different, there are two `SyntaxFactory`s, and everything else is different.  Additionally, C# uses switch blocks while VB.NET uses select blocks.  Finally, none of the other code fixes derived from a base class, so I just copied them.